tools/prowgen/pkg/decorator/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
outside of scope but seems like we can probably remove the bazel stuff at some point...

Reply:
Agreed. It looks these .bazel files are not actually used for critical jobs - since presubmit Prow jobs didn't fail when I didn't update them.